ghc-filesystem is only needed/used for old MacOS systems

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -331,7 +331,7 @@ if(NOT Launcher_FORCE_BUNDLED_LIBS)
     find_package(tomlplusplus 3.2.0 QUIET)
 
     # Find ghc_filesystem
-    find_package(ghc_filesystem QUIET)
+    SET(ghc_filesystem_FOUND YES)
 
     # Find cmark
     find_package(cmark QUIET)
--- a/launcher/CMakeLists.txt
+++ b/launcher/CMakeLists.txt
@@ -1241,7 +1241,7 @@ target_link_libraries(Launcher_logic
     BuildConfig
     Katabasis
     Qt${QT_VERSION_MAJOR}::Widgets
-    ghcFilesystem::ghc_filesystem
+#    ghcFilesystem::ghc_filesystem
 )
 
 if (UNIX AND NOT CYGWIN AND NOT APPLE)
